packages/bot-engine/whatsapp/resumeWhatsAppFlow.ts

```diff
@@ -153,6 +153,8 @@ const getIncomingMessageContent = async ({
         env.NEXTAUTH_URL +
         `/api/typebots/${typebotId}/whatsapp/media/${mediaId}`
       )
+    case 'location':
+      return `Location: ${message.location.latitude},${message.location.longitude}`
```

Comment:
I think it's better to just parse `${message.location.latitude}, ${message.location.longitude}`.

With a space after the comma